contribute/documentation/README.md

```diff
@@ -0,0 +1,234 @@
+
+![Edit a topic](edit-file.png)
+
+1/ Add your changes to the topic.
+
+1/ Scroll to the bottom of the page and enter a branch name.
+   
+   For example, enter `clarified dashboard panel definition`.
+
+1/ Click **Commit**.
+
+   GitHub prompts you to create a PR.
+
+1/ Complete the prompts provided in the body of the PR.
+
+1/ Click **Create pull request**.
+
+### Write a topic
+
+At Grafana Labs, we use the principles of topic-based authoring when we write technical documentation. Topic-based authoring provides guidelines for writing three *types* of technical documentation: concept, task, and reference. Before you begin writing, establish the topic type you want to write.
+
+#### Understanding topic types
+
+Technical content is divided into three topic types: concept, task, and reference.
+
+- **Concept**: A concept topic explains *what* a feature (or idea) is, and why it is important.
+- **Task**: A task topic explains *how* to complete an end user procedure in the system. Task topics contain steps.
+- **Reference** A reference topic contains lookup information that a user might consult when they complete a task. Documenting a list of values with descriptions is a common form of reference topic.
+
+**Example**
+
+Suppose you are writing content for a site called _Doggie handbook_. You might organize your topics like this: 
+
+**Concepts**
+
+- What a dog is
+- Brief history of dogs
+- Why you might want a dog
+- Tasks dogs can be trained to do
+
+**Tasks**
+
+- Feed the dog
+- Groom the dog
+- Train the dog
+
+**References**
+
+- List of dog equipment you will need
+- Table of breeds that includes breed name, size range, short or long hair, and type of dog
+
+#### Prepare your environment
+
+Before you begin writing, we recommend that you fork and clone the Grafana repository so that you can use a text editor locally to create branches, commit your changes, and create a PR.
+
+While this document doesn't include git commands or descriptions of Github operations, you might find these links useful.
+
+- [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git): We store all source code, including documentation, in Git repositories.
+- [Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo): Locate the repo you want to clone, and fork it.
+- [Clone a repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository): Clone the repository to your local machine.
+- [Create a branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging): Before you make change, create a branch. Do not push changes against the `main` branch.
+- [Create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request): After you add, commit, and push your changes, create a PR in Github.
+
+#### Use a documentation template to contribute a topic
+
```

Comment:
@GrafanaWriter ,  do we show folks how to add a directory and `_index.md` or how to add a supporting topic to a topic area that already has an index file?